infrastructure/views/api.py

```diff
@@ -175,7 +178,7 @@ def add_filters(self, qs, params, filter_map):
             if v in params:
                 query_dict[k] = params[v]
 
-        return qs.filter(**query_dict)
+        return qs.filter(**query_dict).distinct()
```

Comment:
Is there test coverage for this?

Happy to ship this change first and ship the test as a second PR.

Reply:
Thanks. there's a wide reaching, end to end test called test_search_with_quarterly that should cover this. The problem was caused by joining the two queries into one and not filtering by distinct. That test will complain if that changes

Yes could add a unit test in addition to that